changelogs/fragments/1933-zos_blockinfile-false_negative_double_quotes.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+   - zos_blockinfile - Previously, double quotes were being handled separately as it was not supported by zoau.
```

Comment:
Text looks good, I would just write zoau in all caps, as ZOAU, since it's a product's name and an initialism.